android/app/build.gradle

```diff
@@ -116,8 +116,7 @@ def getVersionName = { ->
 }
 
 android {
-    compileSdkVersion 24
-    buildToolsVersion "26.0.2"
+    compileSdkVersion 27
```

Comment:
Does that impact our target mobile version?

Reply:
No there is no impact on the target version. :smile: 
From https://medium.com/google-developers/picking-your-compilesdkversion-minsdkversion-targetsdkversion-a098a0341ebd (article by a dev at google):
> It should be emphasized that changing your compileSdkVersion does not change runtime behavior... 
> Therefore it is strongly recommended that you always compile with the latest SDK.